doc/source/whatsnew/v0.25.0.rst

```diff
@@ -672,7 +672,7 @@ Indexing
 - Bug in which :meth:`DataFrame.to_csv` caused a segfault for a reindexed data frame, when the indices were single-level :class:`MultiIndex` (:issue:`26303`).
 - Fixed bug where assigning a :class:`arrays.PandasArray` to a :class:`pandas.core.frame.DataFrame` would raise error (:issue:`26390`)
 - Allow keyword arguments for callable local reference used in the :meth:`DataFrame.query` string (:issue:`26426`)
-
+- Bug which produced ``AttributeError`` on partial matching :class:`Timestamp`s in a :class:`MultiIndex`  (:issue:`26944`)
```

Comment:
I don't think you can do the `s` immediately following the backtick. Probably just remove it.